### DISCUSSION TITLE
I think there should be classes for "default" behaviour

### DISCUSSION THREAD

**Discussion by Billuc** [saadeghi/daisyui#2426] [Ideas / feature requests]
Hi,

I have been using DaisyUI for a few months and I think it's good.
However, there is one thing that is annoying me a bit. It is that, for some components, there is a class to modify the look or behaviour but there is no way to 'cancel' that class.

An example might be better : 
I have a button and I want that button to be a circle on small devices (because I have only an icon), but I want the default look on computer screens (because I have an icon + a text).

The code would look something like this : 
```html
<button class="btn btn-circle md:???"/>
```

If it were the other way (default on small devices and circle on big ones), I could do this : 
```html
<button class="btn md:btn-circle"/>
```

I think it might be useful to have a 'btn-auto' class or something similar to revert to default behaviour/look.
'btn-circle' is an example, there are others I have in mind like 'drawer-end'.

What do you think ?

**Comment by Billuc** [comment]
There is a similar discussion here : #2253

**Comment by saadeghi** [comment]
Tailwind CSS has responsive prefixes like `max-sm:`, `max-md:`, etc.  
So you can simply use `btn max-md:btn-circle`

This approach is better than _reseting_ the style on specific size.

Let me know if you have any questions.

**Reply by Billuc** [reply]
Wow ! I didn't know that ! Thanks !
